packages/extension-api/src/extension-api.d.ts

```diff
@@ -1149,6 +1149,22 @@ declare module '@podman-desktop/api' {
     export const onDidStateChange: Event<boolean>;
   }
 
+  export interface DefaultRegistry {
+    registry: {
+      prefix: string;
```

Comment:
question: can we use `name` and `url` instead of `prefix` and `location` to be consistent with `RegistrySuggestedProvider` below? And then we could pass the registry directly to the suggestion:

<img width="430" height="157" alt="Image" src="https://github.com/user-attachments/assets/dacbe716-b956-4264-979e-a2bde439779b" />